src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp

```diff
@@ -0,0 +1,255 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/GeneralizedHarmonic/Equations.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+
+namespace GeneralizedHarmonic {
+
+template <size_t Dim>
```

Comment:
I find it generally a bit strange that `m` and `n` are used in loops over spatial indices instead of `i, j, k, l`. Is there a good reason for this?

Reply:
In SpEC the convention was to use i,j,k,l for free indices, and m,n,... for contracted indices

Reply:
Okay, well since hopefully TensorExpressions can replace a lot of this let's leave it as is then :)